app/grandchallenge/challenges/admin.py

```diff
@@ -141,3 +252,7 @@ def send_status_update_email(self, request, queryset):
     ChallengeRequestGroupObjectPermission, GroupObjectPermissionAdmin
 )
 admin.site.register(ChallengeSeries)
+
+admin.site.register(
```

Comment:
Not sure about this one, only the Group side of things will be used. But perhaps it is best practice to add a UserObject version as well?

Reply:
I think the group object one is enough